doc/netplan-yaml.md

```diff
@@ -697,6 +702,40 @@ client processes as specified in the Netplan YAML.
     >
     > **Requires feature: `dhcp-use-domains`**
 
+## IPv6 Router Advertisement Overrides
+Overrides for IPv6 Router Advertisement (RA) behaviour (only supported with
+networkd back end).
+
+- **`ipv6-ra-overrides`** (mapping)
```

Comment:
We can rename it to `ra-overrides`

About reusing `accept-ra` directly, the only concern I have is that it would not allow using the `ra-overrides` with `accept-ra` unset (Using the host kernel default setting)